src/plugins/questify/styles.css

```
@@ -0,0 +1,337 @@
+    margin-top: 8px !important;
+}
+
+.questify-form-description {
+    font-size: 14px !important;
+}
+
+.questify-select {
+    border-radius: 3px !important;
+    margin-top: 4px !important;
+    min-height: 45px !important;
+}
+
+.questify-inline-group-item .questify-select {
+    margin-top: 0 !important;
+}
+
+.questify-button,
+.questify-sub-inline-group [class^="swatch"] {
+    height: 38px !important;
+    min-height: 38px !important;
+    width: 80px !important;
+    min-width: 80px !important;
+    padding: 2px !important;
+    border-radius: 3px !important;
+}
+
+.questify-alert-icon {
+    flex: 0 !important;
+    place-content: center center !important;
+    background-color: #212126 !important;
+    border-radius: 3px !important;
+    padding: 0 10px !important;
+    color: #d9d9db !important;
+    border: 1px solid #39393e !important;
+    min-height: 43px !important;
+    display: flex !important;
+    align-items: center !important;
+}
+
+.questify-alert-icon.questify-playing-audio {
+    color: #cc3a40 !important;
+}
+
+.questify-button [class^="contents"] {
+    margin: 0 8px !important;
+}
+
+.questify-button-red {
+    background-color: #913636 !important;
+    color: #fff !important;
+}
+
+.questify-button-red:hover {
+    background-color: #6e2a27 !important;
+}
+
+.questify-button-blue {
+    background-color: #377591 !important;
+    color: #fff !important;
+}
+
+.questify-button-blue:hover {
+    background-color: #2d5568 !important;
+}
+
+#questify-dummy-quest-button {
+    border-left: 5px solid #161619 !important;
+    border-radius: 3px !important;
+}
+
+.questify-text-input,
+.questify-text-area {
+    margin-top: 4px !important;
+    width: 100% !important;
+    border-radius: 3px !important;
+}
+
+.questify-text-area {
+    height: 150px !important;
+}
+
+.questify-color-picker-container {
+    display: grid !important;
+    grid-template-columns: 1fr 1fr !important;
+    gap: 8px !important;
+    margin-top: 8px !important;
+}
+
+.questify-color-picker-group {
+    display: flex !important;
+    flex-direction: column !important;
+}
+
+.questify-color-picker-with-buttons {
+    background-color: #212126 !important;
+    border: 1px solid #39393e !important;
+    border-radius: 3px !important;
+    padding: 8px !important;
+}
+
+.questify-color-picker-with-buttons [class^="swatch"],
+.questify-color-picker-with-buttons [class^="colorSwatch"] {
+    border-radius: 3px !important;
+    width: 100% !important;
+    margin-bottom: 4px !important;
+}
+
+.questify-wide-button {
+    width: 100% !important;
+    flex: 1 !important;
+}
+
+.questify-flex-65 {
+    flex: 6 !important;
+}
+
+.questify-flex-35 {
+    flex: 4 !important;
+}
+
+.questify-inline-code-block {
+    display: inline-block !important;
+    padding: 0 4px !important;
+    border-radius: 3px !important;
+    background-color: #171717 !important;
+    color: #b9bbbe !important;
+    font-family: "Uni Sans", sans-serif !important;
```

```suggestion
    font-family: var(--font-code);
```

use a discord built-in font, like `--font-code`, instead of hard-coding the font. Otherwise, it will look really weird when a custom font is used.